tensorflow/python/estimator/inputs/queues/feeding_functions_test.py

```diff
@@ -290,6 +290,40 @@ def testOrderedDictNumpyFeedFnLargeBatchWithSmallArrayAndMultipleEpochs(self):
     actual = aff()
     self.assertEqual(expected, vals_to_list(actual))
 
+  def testFillArraySmall(self):
+    a = np.ones(shape=[32, 32], dtype=np.int32).tolist() + \
+          np.ones(shape=[32, 36], dtype=np.int32).tolist()
```

Comment:
Please don't use \ continuation, simply indent next line by four (two less than you have)